### PR TITLE
feat(log): log the url (from X-CDN-URL header)

### PR DESCRIPTION
**Pull request by kptdobe**
PR for #3 

Log entry in Coralogix would looks like: 

```
{
   "level"  :  "info" ,
   "request"  : {
     "url"  :  "https://alex.helix-demo.xyz/content/images/car1.png" 
  },
   "ow"  : {
     "activationId"  :  "xyz" ,
     "actionName"  :  "/acapt/helix-services-private/dispatch@3.0.10" ,
     "transactionId"  :  "abc" 
  },
   "message"  :  "A log message" ,
   "timestamp"  :  "2020-03-02T16:41:29.837901214Z" 
}
```

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (tripodsan) [inline .vscode/launch.json]: do you need this in the PR ?

File: src/logger.js [hunk 2]
```diff
@@ -172,6 +180,20 @@ async function wrap(fn, params = {}, { logger = rootLogger, fields = {}, level }
...
+    let url = 'n/a';
+    if (params.__ow_headers && params.__ow_headers['x-cdn-request-id']) {
+      const cdn = params.__ow_headers['x-cdn-request-id'];
+      const index = cdn.indexOf(':');
+      if (index > 0) {
+        url = cdn.substring(index + 1);
+      }
+    }
+
+    CLS_NAMESPACE.set(LOGGER_REQUEST_FIELDS_NAME, {
```
Comment (tripodsan) [inline src/logger.js]: maybe ommit the `request` completely, if `url` is not pressent?

File: src/logger.js [hunk 3]
```diff
@@ -78,14 +79,21 @@ class OpenWhiskLogger extends MultiLogger {
...
-          return {
-            ow,
-            ...fields,
-          };
+          ret.ow = ow;
+        }
+
+        const req = CLS_NAMESPACE.get(LOGGER_REQUEST_FIELDS_NAME);
+        if (req) {
+          ret.request = req;
```
Comment (tripodsan) [inline src/logger.js]: hmm.. I would expect `request` to be what the action receives. what about `cdn-request` ? or at least `request.cnd-url` ?  or `cnd.url` ?

Comment (kptdobe) [reply src/logger.js]: Finding the name was the hardest part of the job :) But I agree, I am really bad for finding names. I am fine with `cdn.url`.

File: .vscode/launch.json (hunk 1 above)
Comment (kptdobe) [reply .vscode/launch.json]: No but it is surprising the file is not there yet. We usually have it everywhere.

**Comment by kptdobe** [comment]
I renamed to `cdn.url` and I played with `git rebase` to create a single commit containing all the changes in one shot.

File: .vscode/launch.json (hunk 1 above)
Comment (tripodsan) [reply .vscode/launch.json]: so....can you delete it ?

Comment (kptdobe) [reply .vscode/launch.json]: Removed.
I do not necessarily agree, the majority of us uses VS Code, this is so convenient to have debugging env ready on project checkout...

**Pull request merged**